src/main/java/com/target/studentinfo/dto/mapper/StudentMapper.java

```diff
@@ -3,13 +3,108 @@
+        return Student.builder()
+                .firstName(studentRequest.getFirstName())
+                .lastName(studentRequest.getLastName())
+                .emailId(studentRequest.getEmailId())
+                .bloodGroup(studentRequest.getBloodGroup())
+                .fatherName(studentRequest.getFatherName())
+                .motherName(studentRequest.getMotherName())
+                .age(studentRequest.getAge())
+                .gender(studentRequest.getGender())
+                .address(studentRequest.getAddress())
+                .standard(studentRequest.getStandard())
+                .extraCurricular(studentRequest.getExtraCurricular())
+                .allergies(studentRequest.getAllergies())
+                .transport(studentRequest.getTransport())
+                .build();
+    }
+
+    public StudentResponse toStudentResponse(Student student) {
+        return StudentResponse.builder()
+                .id(student.getId())
+                .firstName(student.getFirstName())
+                .lastName(student.getLastName())
+                .emailId(student.getEmailId())
+                .bloodGroup(student.getBloodGroup())
+                .fatherName(student.getFatherName())
+                .motherName(student.getMotherName())
+                .age(student.getAge())
+                .gender(student.getGender())
+                .address(student.getAddress())
+                .standard(student.getStandard())
+                .extraCurricular(student.getExtraCurricular())
+                .allergies(student.getAllergies())
+                .transport(student.getTransport())
+                .build();
+    }
+
+    public List<StudentResponse> StudentDetails(List<Student> students) {
+        System.out.println(students);
+        ArrayList<StudentResponse> details = new ArrayList<StudentResponse>();
+        for (Student student : students) {
+            details.add(
+                    StudentResponse.builder()
+                            .id(student.getId())
+                            .firstName(student.getFirstName())
+                            .lastName(student.getLastName())
+                            .emailId(student.getEmailId())
+                            .bloodGroup(student.getBloodGroup())
+                            .fatherName(student.getFatherName())
+                            .motherName(student.getMotherName())
+                            .age(student.getAge())
+                            .gender(student.getGender())
+                            .address(student.getAddress())
+                            .standard(student.getStandard())
+                            .extraCurricular(student.getExtraCurricular())
+                            .allergies(student.getAllergies())
+                            .transport(student.getTransport())
+                            .build()
+            );
+        }
+        return details;
+    }
```

Comment:
Few points
1. Never name a function with capital letters.
2. Make the function name intuitive. For eg, if you return List<StudentResponse>, a good function name would be `toStudentResponses()` or `toStudentResponseList()`. 
3. Please don't rewrite the same functionality again. You can use `toStudentResponse()` inside this function to map each student.

```
    public List<StudentResponse> toStudentResponses(List<Student> students) {
        ArrayList<StudentResponse> studentResponses = new ArrayList<>();
        for (Student student: students) {
            StudentResponse studentResponse = toStudentResponse(student);
            studentResponses.add(studentResponse);
        }
        return studentResponses;
    }
```